backend/handler/email.go

```diff
@@ -138,6 +140,10 @@ func (h *EmailHandler) Create(c echo.Context) error {
 			return fmt.Errorf("failed to create audit log: %w", err)
 		}
 
+		if !h.cfg.Emails.RequireVerification {
+			utils.NotifyUserChange(c, tx, h.persister, events.EmailCreate, userId)
+		}
+
```

Comment:
We also need to check if the user has at least one email, because when a new user registers himself, we get `user.update.email.create` (after email verification is done) which is the wrong event, we want `user.create`. So if a user already has an email trigger `user.update.email.create`, if the user has no email trigger `user.create`.